dev.levz.TinyImageFinder.yml

```diff
@@ -0,0 +1,18 @@
+app-id: dev.levz.TinyImageFinder
+runtime: org.kde.Platform
+runtime-version: '5.15'
```

Comment:
this runtie is EOL for two years.

Reply:
```
FB: Using Flatpak version 1.14.4
FB: Running 'flatpak --user update --subpath= -y --noninteractive org.kde.Sdk/x86_64/5.15' on host
Info: org.kde.Sdk is end-of-life, with reason: We strongly recommend moving to the latest stable version of the Plaform and SDK
Info: org.kde.Sdk.Locale is end-of-life, with reason: We strongly recommend moving to the latest stable version of the Plaform and SDK
Info: org.freedesktop.Sdk.Extension.mono6 is end-of-life, with reason: EOL because of EOL runtime
Info: org.freedesktop.Platform.GL.default is end-of-life, with reason: org.freedesktop.Platform 20.08 is no longer receiving fixes and security updates. Please update to a supported runtime version.
```

Reply:
Fixed to 5.15-22.08.